app/src/main/java/com/example/practiceapp/SearchActivity.kt

```diff
@@ -3,21 +3,22 @@ package com.example.practiceapp
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import androidx.fragment.app.Fragment
-import com.example.practiceapp.databinding.ActivityBuscarBinding
+import com.example.practiceapp.databinding.ActivitySearchBinding
+
 private lateinit var currentFragment : Fragment
-private lateinit var binding: ActivityBuscarBinding
-class BuscarActivity : AppCompatActivity() {
+private lateinit var binding: ActivitySearchBinding
+class SearchActivity : AppCompatActivity() {
```

Comment:
Why is this Activity called `SearchActivity`?

Reply:
Thanks for the advice.